aten/src/ATen/test/Vec256Test.cpp

```diff
@@ -0,0 +1,1091 @@
+#include <ATen/test/Vec256Test.h>
```

Comment:
Can you please combine with `aten/src/ATen/test/vec256_test.cpp` as there are look similar and confusing to have names like `vec256_test` and `Vec256Test`. It is fine if you will leave all duplicate functionality tests from vec256 as yours seems to provide better coverage.

Reply:
it is good. 
it has a small typo: https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/test/vec256_test.cpp#L110-L111

Reply:
its using tensor. we can ask its author to combine. I believe that style of testing can be added to this one.

Reply:
I agree that a combination of these two tests might be postponed and done later, however in this case please rename your files into something like `vec256_test_all_types.cpp`